src/sentry/integrations/vsts/client.py

```diff
@@ -204,30 +225,31 @@ def get_commit_range(self, instance, repo_id, start_sha, end_sha):
             },
         )
 
-    def get_project(self, instance, project_id):
+    def get_project(self, instance: str, project_id: str) -> Response:
         return self.get(
             VstsApiPath.project.format(instance=instance, project_id=project_id),
             params={"stateFilter": "WellFormed"},
         )
 
-    def get_projects(self, instance):
-        def gen_params(page_number, page_size):
-            # ADO supports a continuation token in the response but only in the newer API version (https://docs.microsoft.com/en-us/rest/api/azure/devops/core/projects/list?view=azure-devops-rest-6.1)
-            # the token comes as a response header instead of the body and our API clients currently only return the body
-            # we can use count, $skip, and $top to get the same result
+    def get_projects(self, instance: str) -> Response:
+        def gen_params(page_number: int, page_size: int) -> Mapping[str, Union[str, int]]:
+            # ADO supports a continuation token in the response but only in the newer API version (
+            # https://docs.microsoft.com/en-us/rest/api/azure/devops/core/projects/list?view=azure-devops-rest-6.1
+            # ). The token comes as a response header instead of the body and our API clients
+            # currently only return the body we can use count, $skip, and $top to get the same result.
             offset = self.page_size * page_number
             return {"stateFilter": "WellFormed", "$skip": offset, "$top": page_size}
 
-        def get_results(resp):
-            return resp["value"]
+        def get_results(resp: Response) -> Sequence[Any]:
+            return resp["value"]  # type: ignore
```

Comment:
Why the ignore here? Could you not assert the type?

Reply:
Good idea, I'll update this.